src/openrct2-ui/windows/Staff.cpp

```diff
@@ -886,9 +886,11 @@ void window_staff_options_invalidate(rct_window* w)
             w->pressed_widgets &= ~((1 << WIDX_CHECKBOX_1) | (1 << WIDX_CHECKBOX_2));
             w->pressed_widgets |= peep->StaffOrders << WIDX_CHECKBOX_1;
             break;
-        case STAFF_TYPE_SECURITY:
+        case StaffType::Security:
             // Security guards don't have an options screen.
             break;
+        case StaffType::Count:
```

Comment:
Just add a default. Staff Type Count should never be an option in a switch

Reply:
I asked for th removal of default because it needs to use all enumerations otherwise it errors out, don't you think it looks clearer? This way we can see what actually is being handled or not